app/controllers/sufia/resource_sync_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Sufia::ResourceSyncController < ApplicationController
+  def source_description
+    host = request.host
+    # Caching based on host, for multitenancy support
+    body = Rails.cache.fetch("source_description_#{host}", expires_in: 1.week) do
```

Comment:
Yes, that's true, but I don't think assigning the extra variable makes it more readable. It think it's the opposite, in fact, because now I have to mentally keep track of `host`. Whereas using the provided `request` object seems to be the Rails way of using what's provided to us, and doesn't extend the line to any extraordinary length beyond what it already is.

You also do this three times in the class, once for each public method, which to me just yells "refactor."